src/react/LoginPage.tsx

```diff
@@ -275,43 +272,46 @@ const RightColumn = ({
       {loginType === 'oauth' ? (
         <OAuthForm
           defaultRealm={defaultRealm}
+          defaultOAuthURL={authOptions?.oauth?.oauth_api_url || ''}
```

Comment:
Here `authOptions?.oauth` is always defined since we're checking `loginType === 'oauth'`. Any ideas on how to improve typings?

Reply:
I didn't want to create a config provider here considering the future updates on the next development cycle. This is the fastest way to get it implemented.